README.md

```diff
@@ -9,6 +9,18 @@ The process for submitting an RFC is documented in the [RFC Process](CONTRIBUTIN
 
 For the main reference implementation of the ILP stack, see [Interledger.js](https://github.com/interledgerjs).
 
+## ASN.1 and OER
+
+A number of the protocols define data structures in ASN.1 notation. The collection of definitions is in [asn1](./asn1).
+
+Changes to these files are automatically checked and compiled during CI using online ASN.1 tools from OSS Nokalva. If you need ASN.1 tools for any work you're doing on Interledger please contact them for assistance.
+
+[![OSS Nokalva](https://raw.githubusercontent.com/interledger/rfcs/master/assets/osslogo.png)](http://asn1-playground.oss.com/)
```

Comment:
The image doesn't show up. `![GitHub Logo](/images/logo.png)` is the format so I think it might be because of the first `[`

Reply:
It's because it will only be uploaded as part of this PR

Reply:
You sure about that leading bracket though?

Reply:
Yes, it's a link around an image